fhir-install/src/main/docker/ibm-fhir-term-graph-loader/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# ----------------------------------------------------------------------------
+# (C) Copyright IBM Corp. 2021
+#
+# SPDX-License-Identifier: Apache-2.0
+# ----------------------------------------------------------------------------
+
+# Build stage
+FROM maven:3.6.0-jdk-11-slim AS build
+COPY . /
+
+RUN mvn -B org.apache.maven.plugins:maven-dependency-plugin:3.1.2:go-offline -f /fhir-examples -DexcludeReactor=true -Dmaven.wagon.http.retryHandler.count=3
+RUN mvn -B org.apache.maven.plugins:maven-dependency-plugin:3.1.2:resolve-plugins -f /fhir-examples -DexcludeReactor=true -Dmaven.wagon.http.retryHandler.count=3
+RUN mvn -f /fhir-examples/pom.xml install -DskipTests=true
+
+RUN mvn -B org.apache.maven.plugins:maven-dependency-plugin:3.1.2:go-offline -f /fhir-parent -DexcludeReactor=true -Dmaven.wagon.http.retryHandler.count=3
+RUN mvn -B org.apache.maven.plugins:maven-dependency-plugin:3.1.2:resolve-plugins -f /fhir-parent -DexcludeReactor=true -Dmaven.wagon.http.retryHandler.count=3
+RUN mvn -f /fhir-parent/pom.xml install -DskipTests=true -pl ../fhir-term-graph -am
+
+RUN mvn org.apache.maven.plugins:maven-dependency-plugin:3.2.0:copy-dependencies -f /fhir-parent -pl ../fhir-term-graph -am -DoutputDirectory=target/ftg
```

Comment:
@prb112 I wonder if we should consider about adding a fully dockerized maven build like this to our other images...  if we did it, then we could more easily use something like Docker (or Quay) automated builds to keep our images fresh... (which could help with #1412)

Reply:
yep - talked with him about this - it's complicated currently